Rajkumar562/task2.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style> 
+div {
+position: absolute;
+padding: 5px;
+width: 100px;
+height: 100px;
+}
+.retro{
+  height:100%;
+  width:100%;
+}
+.text {
+  text-align:center;
+  bottom:100px;
+  width: 200px;
+  height: 80px;  
+  padding: 10px;
+  border-radius: 10px;
+  font-weight:bold;
+  color:bold black;
+  font-family:roboto,sans-serif;
+  background-color: rgb(255, 144, 101);
+  box-shadow:10px 10px rgb(192, 67, 197);
+  box-sizing: border-box;
+  bottom:50px;
+right: 50px;
+}
+</style>
+</head>
+<body>
+  <div></div>
+  <img class="retro" src="https://github.com/CC-BHU/web-development/raw/main/assets/images/task2.jpg" alt="retro city img">
+<div class="text">Welcome to the Retro City, looks mesmerising isn't it?</div>

```

Comment:
This text-box isn't relative to the image, try adding all this to a parent div that have `position: absolute;`.